components/ServicePage/ServicesItem.tsx

```diff
@@ -0,0 +1,83 @@
+import styles from "@/styles/ServicesItem.module.css";
+import Button from "../base/Button";
+import { fontSyne, fontSatoshi, fontRobotoMono } from "@/pages/_app";
+
+type serviceItemProps = {
+  service_name?: string;
```

Comment:
Do not make every props optional, reason:
1. I think our page will not have the duplicate service. So each name of theirs should be unique.
2. You do not have one with default value, plz refer to button.
![image](https://user-images.githubusercontent.com/73293255/225846369-4ce1a252-7f7b-4be0-b9e2-2c1bfdfec9d2.png)

Reply:
For this component, it does not make sense to have all props optional. I agree with cam. If all props are optional, what would you like to show on the page with this component?

However, it is okay for some rare conditions to have all optional props even with no default value. But it is not the case for this component.

Reply:
thanks, due to the name, introduction, and heading of each service will not be repeated, I set service_heading, service_info1, service_info2, and service_name to not optional, and set the default icon and image